week-3-nestjs-back-end/w03e01-modular-app/src/restaurants/restaurants.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from '@nestjs/common';
+import { CreateRestaurantDto } from './dto/create-restaurant.dto';
+import { UpdateRestaurantDto } from './dto/update-restaurant.dto';
+import { Restaurant } from './entities/restaurant.entity';
+import { restaurants } from './resource';
+
+@Injectable()
+export class RestaurantsService {
+  private restaurants = restaurants;
+  create(createRestaurantDto: CreateRestaurantDto) {
+    const restaurant = new Restaurant();
+    restaurant.id = Math.round(Math.random() * 100);
```

Comment:
Zobacz, skoro zrobiłeś `Restaurant` jako `class` (entity).
To warto z tego skorzystać.

Na razie to `entity` jest "anemiczne" - nic tam nie ma poza kształtem danych.
A może warto skorzystać z konstruktora i przekazać tam `DTO` i to właśnie tam wpisywać te dane.

Najlepiej korzystając z destrukturyzacji.